src/plugins/prefs/autoconfig/firefox.cfg

```diff
@@ -0,0 +1,47 @@
+//
+
+const configName = getPref("elektra.config.file");
```

Comment:
what about checking for return value and pop up a message that the setup scripts should be runned?